crates/biome_cli/tests/snapshots/main_cases_overrides_organize_imports/does_handle_included_file_and_disable_organize_imports.snap

```diff
@@ -24,7 +23,7 @@ import { lorem, foom, bar } from "foo";
 ## `test.js`
 
 ```js
-import { bar, foom, lorem } from "foo";
+import { bar , foom, lorem} from "foo";
```

Comment:
I see you mentioned the trivia handling in the description of the PR already, but how hard would it be to fix this?

Reply:
Technically I already implemented the fix. However, The code does not satisfy me. I have to try another approach.

Reply:
It is now fixed for imports.
I will port the change to exports and imports attributes.